### PR TITLE
feat: grid-list added

### PR DESCRIPTION
**Pull request by deekshithmd**
Added grid and list component for the component library

### CODE REVIEW

**Review by kushalsourav — APPROVED** [review]
looks good to merge

File: components/list/list.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    padding: 0;
+}
+
+.stacked-container li{
+    padding: 1rem;
+    background-color: var(--success-background);
+    border: 1px solid var(--border-black);
+    font-size: 1.5rem;
+    width: 100%;
+}
```
Comment (mayuresh-tech) [inline components/list/list.css]: Maybe you can keep some spacing vertically between notifications

Comment (mayuresh-tech) [reply components/list/list.css]: Else everything looks good!

File: components/grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    font-size: 2rem;
+}
+
+.row-3{
+    grid-area: row-3;
+    background-color: var(--secondary-color);
+    padding: 2rem;
+    text-align: center;
+    font-size: 2rem;
+}
```
Comment (m26unkwn) [inline components/grid/grid.css:105]: these   grid classes naming and representation is great.

File: components/grid/grid.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            <span class="row-2">1/2</span>
+        </div>
+        <hr>
+        <!--3 row grid-->
+        <div class="grid-row-3">
+            <span class="row-1">1/3</span>
+            <span class="row-2">1/3</span>
+            <span class="row-3">1/3</span>
+        </div>
+</body>
```
Comment (m26unkwn) [inline components/grid/grid.html:57]: Documentation of grid is also good.

**Review by m26unkwn — APPROVED** [review]
Everything is seems great.

**Review by PuneetP16 — APPROVED** [review]
Looks good to merge.

File: components/list/list.css (hunk 1 above)
Comment (deekshithmd) [reply components/list/list.css]: Ok, Thank you for review

File: components/grid/grid.css (hunk 2 above)
Comment (deekshithmd) [reply components/grid/grid.css:105]: Thank you for review

File: components/grid/grid.html (hunk 3 above)
Comment (deekshithmd) [reply components/grid/grid.html:57]: Thank you for review

**Comment by deekshithmd** [comment]
> Looks good to merge.

Thank you for review

**Comment by deekshithmd** [comment]
> looks good to merge

Thank you for review

**Pull request merged**